pkg/virt-controller/services/template_test.go

```diff
@@ -203,6 +203,32 @@ var _ = Describe("Template", func() {
 				Expect(value).To(Equal("default,test1"))
 			})
 		})
+		Context("with tungsten fabric annotation", func() {
+			It("should add tungsten fabric networks in the pod annotation", func() {
+				vmi := v1.VirtualMachineInstance{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "testvmi",
+						Namespace: "default",
+						UID:       "1234",
+					},
+					Spec: v1.VirtualMachineInstanceSpec{
+						Domain: v1.DomainSpec{},
+						Networks: []v1.Network{
+							{Name: "{\"domain\":\"default-domain\", \"project\": \"k8s-default\",\"name\":\"bla\"}",
```

Comment:
For this unit test, the 'Name' value should be different than the 'NetworkName' value. That ensures we don't accidentally inject the 'Name' value as the TungstenFabric's annotation value instead of NetworkName.

Reply:
done